ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -95,12 +99,37 @@
                     }, parseInt(autoSaveInterval) * 1000);
                 }
             }
+            this._model.subscribe((action) => {
+                const { payload } = action;
+                const { changes, field } = payload;
+                const { items } = field;
+                if (changes && changes.length > 0) {
+                    changes.forEach((change) => {
+                        switch (change.propertyName) {
+                            case "activeChild":
+                                FormContainerV2.hamburgerMenuInstance.handleActiveItem(field);
+                                break;
+                            case "items":
+                                FormContainerV2.hamburgerMenuInstance.handleItemsChange(change, items);
+                                break;
+                            case "visible":
+                                FormContainerV2.hamburgerMenuInstance.updateAttribute(field.id, 'data-cmp-visible', field.visible);
+                                break;
+                            case "enabled":
+                                FormContainerV2.hamburgerMenuInstance.updateAttribute(field.id, 'data-cmp-enabled', field.enabled);
+                                break;
+                        }
+                    });
+                }
+            }, "fieldChanged");
         }
     }
 
     async function onDocumentReady() {
         const startTime = new Date().getTime();
         let elements = document.querySelectorAll(FormContainerV2.selectors.self);
+        const hamburgerMenuInstance = new HamburgerMenu();
```

Comment:
You should initialise inside form container, since there could be multiple form container in a page

Reply:
@rismehta We can not init the hamburger menu inside the FormContainer because we can only initialize after the subscriber is added to form container.